docs/examples/resampling.ipynb

```diff
@@ -0,0 +1,218 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "xds.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Resampling\n",
+    "\n",
+    "In order to copy `rasterio`'s behavior stated [here](https://rasterio.readthedocs.io/en/latest/topics/resampling.html), you can:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from rasterio.enums import Resampling\n",
+    "\n",
+    "# Reproject   \n",
+    "upscale_factor = 2\n",
+    "new_width = xds.rio.width * upscale_factor\n",
+    "new_height = xds.rio.height * upscale_factor\n",
+    "resampling=Resampling.bilinear\n",
+    "\n",
+    "xds_upsampled = xds.rio.reproject(xds.rio.crs, shape=(new_height, new_width))\n"
```

Comment:
Don't forget to pass in `resampling` kwarg.

Reply:
Yes my bad !

Reply:
Thanks for addressing that :+1: